src/scalarstats.jl

```diff
@@ -477,9 +475,8 @@ function crossentropy{T<:Real}(p::AbstractArray{T}, q::AbstractArray{T})
     return -s
 end
 
-function crossentropy{T<:Real}(p::AbstractArray{T}, q::AbstractArray{T}, b::Real)
-    return crossentropy(p,q) / log(b)
-end
+crossentropy(p::AbstractArray{T}, q::AbstractArray{T}, b::Real) where {T<:Real} =
```

Comment:
Is there a rule to decide when to use `{}` and when to omit it? Looks like kind of inconsistent.